src/views/Home.tsx

```diff
@@ -0,0 +1,47 @@
+import { INPUT_VALUE_KEY } from 'appConstants';
+import { Cards, SearchBar } from 'components';
+import { fetchDetailedPokemons } from 'utils';
+import React, { ChangeEvent } from 'react';
+import { PokemonDetails } from 'types';
+import * as Styled from './styled';
+
+interface State {
+  pokemons: Array<PokemonDetails>;
+  inputValue: string;
+  isLoading: boolean;
+}
+
+export class Home extends React.Component<unknown, State> {
+  state: State = {
+    isLoading: true,
+    inputValue: '',
+    pokemons: [],
+  };
+  componentDidMount() {
+    const value = window.localStorage.getItem(INPUT_VALUE_KEY);
+    if (value != null) {
+      this.setState({ inputValue: value });
+    }
+    fetchDetailedPokemons()
+      .then((pokemons) => this.setState({ pokemons }))
```

Comment:
catch?

Reply:
fetchDetailedPokemons function has catch itself so error will be logged. Event if fetchDetailedPokemons catech error, finally empty array will be returned